modules/internal/ChapelDistribution.chpl

```diff
@@ -621,8 +648,41 @@ module ChapelDistribution {
 
     sd._value.dsiBulkAdd(arr, true, true, false);
   }
+
+  // TODO: Implement bulkRemove
+  proc -=(ref sd: domain, inds: [] sd.idxType) where isSparseDom(sd) &&
+    sd.rank==1 {
+
+    if inds.size == 0 then return;
```

Comment:
I don't think these returns are necessary. If the arrays or domains are empty, the loops won't do anything.